modules/custommutator/bech32.h

```diff
@@ -44,6 +44,10 @@ enum CharLimit : size_t {
  *  assertion error. Encoding must be one of BECH32 or BECH32M. */
 std::string Encode(Encoding encoding, const std::string& hrp, const std::vector<uint8_t>& values);
 
+/** Encode a Bech32 or Bech32m string without the checksum. If hrp contains uppercase characters, this will cause an
+ *  assertion error. Encoding must be one of BECH32 or BECH32M. */
+std::string EncodeNoChecksum(const std::string_view hrp, const std::vector<uint8_t>& values);
```

Comment:
Nit:  `string_view` already prevents mutation of the string it points to.  The `const` here only prevents us from reassigning `str` to a different string.  Usually we omit such consts in parameters (here and elsewhere).

```suggestion
std::string EncodeNoChecksum(std::string_view hrp, const std::vector<uint8_t>& values);
```

Reply:
Fixed!